ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py

```diff
@@ -208,7 +208,7 @@
 short_description: Manage the HPE Nimble Storage volume collections
 version_added: "1.0.0"
 notes:
-  -  check_mode is not supported.
+  - This module does not supports C(check_mode).
```

Comment:
This module does not support C(check_mode). (no ending "s" on support). I've not seen the C() annotation before but I guess that's ansible specific?

Reply:
yes that is ansible specific.. that's fine

Reply:
ok will remove 's'